Update for GCC-6-20160221

belka-ew opened the pull request:
@ibuclaw. Since GCC 6-20160131 blank lines are not allowed in the
output by default. See gcc/testsuite/ChangeLog.
99bottles.d generates empty lines after every verse and these tests
fails with "100 blank line(s) in the output". I set
"dg-allow-blank-lines-in-output { 1 }" that restores the behavior of
the previous versions. But I'm not sure if I put this into the right
place.
I'm not ready with this pull request. I will try in the next days to update master to the latest GCC snapshot.

belka-ew committed:
Update for GCC-6-20160131

@ibuclaw. Since GCC 6-20160131 blank lines are not allowed in the
output by default. See gcc/testsuite/ChangeLog.
99bottles.d generates empty lines after every verse and these tests
fails with "100 blank line(s) in the output". I set
"dg-allow-blank-lines-in-output { 1 }" that restores the behavior of
the previous versions. But I'm not sure if I put this into the right
place.

belka-ew committed:
Fix build_common_tree_nodes call

Remove build_common_tree_nodes' second parameter that doesn't exits
anymore.

belka-ew committed:
Fix a minor typo in the testsuite comments

belka-ew committed:
Update the GCC version to 6-20160221

Now GCC is up to date and GDC passes the tests.

belka-ew commented:
Now GCC is up to date and GDC passes the tests.

ibuclaw commented on gcc/d/d-lang.cc:
```diff
@@ -252,8 +252,8 @@ d_init()
-  // Parameters are (signed_char = false, short_double = false).
-  build_common_tree_nodes (false, false);
+  // Parameter (signed_char = false) specifies whether char is signed.
+  build_common_tree_nodes (false);
```
This needs a ChangeLog entry.

belka-ew committed:
Add a ChangeLog entry about gcc-6-20160214 changes

d-lang.cc (d_init): gcc-6-20160214 removed the second parameter
(short_double = false) of build_common_tree_nodes.

belka-ew commented:
Oh sorry; I forgot about the changelog

ibuclaw commented on gcc/d/ChangeLog:
```diff
@@ -1,3 +1,8 @@
+2016-02-22  Eugene Wissner <belka@caraus.de>
+
+	* d-lang.cc (d_init): gcc-6-20160214 removed the second parameter
+	(short_double = false) of build_common_tree_nodes.
```
Don't need to reference the gcc snapshot timestamp.  The message should be simple and boring, with a complete lack of reasoning behind the change (this is what commit messages are for).

ibuclaw replied:
E.g:

```
* d-lang.cc (d_init): Remove short_double parameter from
build_common_tree_nodes.
```

belka-ew commented:
Ok. should I fix the entry?

ibuclaw commented:
Yes please.

belka-ew committed:
Fix the last ChangeLog entry

belka-ew commented:
Done

ibuclaw commented:
OK, I'll work out what to do for `dg-allow-blank-lines-in-output` later.
